src/services/datahub-client.doubles.ts

```diff
@@ -138,4 +166,31 @@ export class HardcodedDatahubClient implements DatahubClient {
     //  error: new Error('Dummy error'),
     //};
   }
+
+  public async getDailyVisitsByPeriod({
+    domainName,
+    dateFrom,
+  }: {
+    domainName: string;
+    dateFrom: Date;
+  }): Promise<DailyVisitsResult> {
+    console.log('getDailyVisitsByPeriod', { domainName, dateFrom });
+    await timeout(1000);
+    const dailyVisits = fakeDailyVisitsData.map((x) => ({
+      periodNumber: x.periodNumber,
+      from: x.from,
+      to: x.to,
+      quantity: x.quantity,
+    }));
+
+    return {
+      success: true,
+      value: dailyVisits,
+    };
+
+    //return {
```

Comment:
Perhaps this should have a comment about what scenario is?

Reply:
mm in all methods have only the response when have and error. You said add a comment for error type? there is an unexpected error

Reply:
I think that the example response meaning is very clear.

And we know why are we using these comments in the doubles, so, in this case, personally, I am fine without any clarificatory comment here.